dockerfiles/Dockerfile.ci

```diff
@@ -1,5 +1,5 @@
 # CI test image for unit/lint/type tests
-FROM node:10-alpine
```

Comment:
We should only use even numbered node releases. We may want to jump to 12 now though.

Reply:
https://medium.com/@nodejs/october-brings-node-js-10-x-to-lts-and-node-js-11-to-current-ae19f8f12b51